package.json

```diff
@@ -0,0 +1,155 @@
+{
+  "name": "example-typescript-template",
+  "version": "0.0.1",
+  "description": "Example Typescript Template",
+  "repository": "git@github.com:brightonsbox/example-typescript-template.git",
+  "license": "MIT",
+  "scripts": {
+    "copy-views": "cp -R server/views dist/server/",
+    "compile-sass": "sass --no-source-map --load-path=node_modules/govuk-frontend ./assets/sass/application.sass:./assets/stylesheets/application.css ./assets/sass/application-ie8.sass:./assets/stylesheets/application-ie8.css --style compressed",
+    "watch-ts": "tsc -w",
+    "watch-views": "nodemon --watch server/views -e html,njk -x npm run copy-views",
+    "watch-node": "DEBUG=gov-starter-server* nodemon --watch dist/ dist/server.js | bunyan -o short",
+    "watch-sass": "npm run compile-sass -- --watch",
+    "build": "npm run compile-sass && tsc && npm run copy-views",
+    "start:dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node\" \"npm run watch-sass\"",
+    "start-feature": "export $(cat feature.env) && node $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
+    "watch-node-feature": "export $(cat feature.env) && nodemon --watch dist/ $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
+    "start-feature:dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node-feature\" \"npm run watch-sass\"",
+    "record-build-info": "node ./bin/record-build-info",
+    "lint": "eslint . --cache --max-warnings 200",
+    "typecheck": "tsc",
+    "test": "jest --runInBand",
```

Comment:
Do all tests need to be run in series? I like to configure jest so that unit tests can run in parallel and integration tests run in series if data they rely on mutating test data.